UnityProject/Assets/Scripts/Sprite Handler/SpriteHandler.cs

```diff
@@ -258,7 +258,7 @@ private void SetSprite(SpriteInfo animationStills)
 
 	public void ChangeSprite(int newSprites)
 	{
-		if (spriteData.List != null)
+		if (spriteData != null && spriteData.List != null)
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
SpriteHandler.ChangeSprite (System.Int32 newSprites) (at /github/workspace/UnityProject/Assets/Scripts/Sprite Handler/SpriteHandler.cs:261)
ClothingItem.RefreshFromClothing (ClothingV2 clothing) (at /github/workspace/UnityProject/Assets/Scripts/Items/ClothingItem.cs:141)
ClothingV2.RefreshClothingItem () (at /github/workspace/UnityProject/Assets/Scripts/Clothing/ClothingV2.cs:217)
ClothingV2.LinkClothingItem (ClothingItem clothingItem) (at /github/workspace/UnityProject/Assets/Scripts/Clothing/ClothingV2.cs:197)
ClothingItem.SetReference (UnityEngine.GameObject Item) (at /github/workspace/UnityProject/Assets/Scripts/Items/ClothingItem.cs:116)
PlayerAppearanceMessage.Process () (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/PlayerAppearanceMessage.cs:47)
GameMessageBase.Process (Mirror.NetworkConnection sentBy) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:25)
GameMessageBase.PreProcess (Mirror.NetworkConnection sentBy, GameMessageBase b) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:18)
Mirror.MessagePacker+<>c__DisplayClass6_02[T,C].<MessageHandler>b__0 (Mirror.NetworkConnection conn, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/MessagePacker.cs:146) Mirror.NetworkConnection.InvokeHandler (System.Int32 msgType, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:219) Mirror.NetworkConnection.TransportReceive (System.ArraySegment1[T] buffer, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:272)
Mirror.ULocalConnectionToServer.Update () (at /github/workspace/UnityProject/Assets/Mirror/Runtime/LocalConnections.cs:110)
Mirror.NetworkClient.Update () (at /github/worksp